### ISSUE TITLE
Don't display the markers from other maps

### ISSUE THREAD

**Issue by dmitriyKharlashin** [CodeCabin/wp-google-maps#1011]
### Steps to reproduce:
- install plugin version _8.0.17_
- create few map entities with few markers and polylines each
- attach map shortcodes to different pages
- at the map, you can see 5 markers and 3 polylines from the attached map, **_as expected_**
![image](https://user-images.githubusercontent.com/33868308/111606204-a6279200-87df-11eb-8248-2be9461e1746.png)
- update the plugin to version _8.1.11_
-  check the page with a map
- I see all markers from all maps
![image](https://user-images.githubusercontent.com/33868308/111607289-c60b8580-87e0-11eb-9554-7b92e7ead129.png)

Is any way do not to display not-needed markers on the map?

**Comment by DylanCodeCabin** [comment]
Thank you for your report. I am sorry this slipped by us! 

The issue should have been resolved from the time it was logged. We are cleaning up this repo in an attempt to reorganize and open a line of communication with reporters. 

As a result, this issue is being closed as `stale` - You can learn more about the future of this repo here:
- https://github.com/CodeCabin/wp-google-maps/issues/1015

Basically, we will be removing all issues, moving relevant items to our internal V9 repo and solving them further there. Once V9 is released to the public, this repo will become more active with issue tracking and enhancement requests
